webapp/components/ContributionForm/ContributionForm.vue

```diff
@@ -169,6 +169,16 @@ export default {
     },
     updateEditorContent(value) {
       // this.form.content = value
+      this.disabled = true
+      if (
+        value.replace(
+          /<p>|<\/p>|<h3>|<\/h3>|<h4>|<\/h4>|<ul>|<\/ul>|<ol>|<\/ol>|<li>|<\/li>|<blockquote>|<\/blockquote>|<strong>|<\/strong>|<em>|<\/em>|<a.*>|<\/a>/gm,
```

Comment:
Please don't use a regex. If you want to strip HTML from a string, the [internet has the answer](https://stackoverflow.com/questions/5002111/how-to-strip-html-tags-from-string-in-javascript).. Regexes are horrendously hard to maintain. "If you found out you can solve your problem with a regex, you have a new problem."

Reply:
to count the absolute characters, a lenght function is not enough. because the editor just inserts these html tags. so i only count the real characters that a user writes. nothing will be deleted or added from the original text. 

since I hold myself back from inserting npm packete. 

or I don't know if this already exists. 

since i love regular expressions that was my solution :)

Reply:
no the number of characters in a content is not limited . i would suggest 2000 characters. maybe 4000. you can say badly .. some like to write whole romanes. maybe there should be a BLOG corner for every user for something like that. 
i think 2000 characters is ok. those are 2 A4 pages

Reply:
replace("<[^>]*>", "")  is the short code ;)

Reply:
the short code is: .replace(/<\/?[^>]+(>|$)  ;)